aws/docker-pull.sh

```diff
@@ -15,6 +15,7 @@ IMAGES=(
   mozilla/syncserver
   mozilla/fxa-profile-server
   mozilla/pushbox
+  rfkelly/channelserver
```

Comment:
Using my own docker image, pending CircleCI config magic over in https://github.com/mozilla-services/pairsona/pull/18